src/main/java/Duke/Task.java

```diff
@@ -0,0 +1,58 @@
+package Duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Costructor method for Task.
+     *
+     * @param description description of a task to be added.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Checks status of task.
+     *
+     * @return 1 if it is done, 0 if not done.
+     */
+    public char isDone() {
```

Comment:
I think the name of this method might be a bit misleading; might lead one to believe that it returns a boolean. Perhaps the name could be changed to more clearly reflect this?